jekyll/_cci2/runner-installation.adoc

```diff
@@ -0,0 +1,154 @@
+
+===== For macOS x86_64
+
+`platform=darwin/amd64`
+
+And then run the following steps to download, verify and install the binary.
+
+```sh
+prefix=/opt/circleci
+sudo mkdir -p "$prefix/workdir"
+base_url="https://circleci-binary-releases.s3.amazonaws.com/circleci-launch-agent"
+echo "Determining latest version of CircleCI Launch Agent"
+agent_version=$(curl "$base_url/release.txt")
+echo "Using CircleCI Launch Agent version $agent_version"
+echo "Downloading and verifying CircleCI Launch Agent Binary"
+curl -sSL "$base_url/$agent_version/checksums.txt" -o checksums.txt
+file="$(grep -F "$platform" checksums.txt | cut -d ' ' -f 2)"
+file="${file:1}"
+mkdir -p "$platform"
+echo "Downloading CircleCI Launch Agent: $file"
+curl --compressed -L "$base_url/$agent_version/$file" -o "$file"
+echo "Verifying CircleCI Launch Agent download"
+sha256sum --check --ignore-missing checksums.txt && chmod +x "$file"; sudo cp "$file" "$prefix/circleci-launch-agent" || echo "Invalid checksum for CircleCI Launch Agent, please try download again"
+```
+
+=== Platform-specific instructions
+
+Please refer to the platform-specific installation instructions:
+
+- linux
+- macOS
+
+=== Configuration file reference
+
+A YAML file is used to configure the Launch Agent; how it communicates with our servers and how it will launch the Task Agent.
+
+The configuration file uses the following format, the parameters are explained in more detail below:
+
+```sh
+api:
+  auth_token: AUTH_TOKEN
+runner:
+  name: RUNNER_NAME
+runner.name
+```
+
+This is a unique name assigned to this particular running Launch Agent, we recommend using the hostname of the machine. It will be used to identify the agent when viewing statuses and job results in the CircleCI UI.
+
+==== api.auth_token
+
+This is a token used to identify the Launch Agent to CircleCI, it will be provided by your Customer Success Manager. An existing token may be shared among many installations, but only allows a particular resource_class to be specified.
+
+==== runner.command_prefix
+
+This allows you to customize how the Task Agent process is launched, our example uses the launch-task script provided below.
+
+==== runner.working_directory
+
+This allows you to control the default working directory used by each job. If the directory already exists, Task Agent will need permissions to write to it. If the directory does not exist, then the Task Agent will need permissions to create it. If `%s` is present in the value, it will be replaced with a different value for each job. These directories will not be automatically removed.
+
+==== runner.cleanup_working_directory
+
+This allows for control of working directory cleanup after each job. The default value is `false`.
+
+==== runner.max_run_time
+
+This can be used to override the default maximum duration the Task Agent will run each job. The value is a string with the following unit identifiers `h`, `m` or `s` for hour minute and seconds respectively:
```

Comment:
```suggestion
This can be used to override the default maximum duration the Task Agent will run each job. The value is a string with the following unit identifiers `h`, `m` or `s` for hour minute and seconds respectively. Consider these valid examples:
```

Reply:
If you commit this suggestion, you will also need to remove line 146 ( `A few valid examples`)

Reply:
Fixed